go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/ceph/ceph-csi
 
-go 1.22.5
+go 1.22.7
+
+toolchain go1.22.9
```

Comment:
Is the toolchain addition required?

Reply:
was added by `go mod tidy` removed it now.